docs/src/doc/user-guide/classes.md

```diff
@@ -7,22 +7,6 @@ class RotatingCube: Node3D() {
 }
 ```
 
-## Registration files
-
-For each class you register, a corresponding registration file is generated (a `gdj` file). These are the files you attach to nodes.
-
-### Location
-
-By default, these files are generated into a folder called `gdj` in the root of your project.
-
-You can however configure the destination folder inside your `build.gradle.kts`:
-
-```kotlin
-godot {
-    registrationFileBaseDir.set(<folder>)
-}
-```
-
```

Comment:
I simply removed it because it doesn't make sense to me to have this part in the class section. The same applies to having a lino to the api difference section. The way you handle your classes has nothing to do with registration and files.